helper/tlsutil/generate.go

```diff
@@ -81,10 +88,28 @@ type CertOpts struct {
 	ExtKeyUsage []x509.ExtKeyUsage
 }
 
+// IsEmpty checks whether any of CAOpts parameters have been populated with
+// non-default values.
+func (c CAOpts) IsEmpty() bool {
```

Comment:
Is this function name accurate? There are a number of parameters which can be non-default and this would return true.